test/common/stats/thread_local_store_test.cc

```diff
@@ -189,6 +246,251 @@ TEST_F(StatsThreadLocalStoreTest, BasicScope) {
   // Includes overflow stat.
   EXPECT_CALL(*this, free(_)).Times(5);
 }
+TEST_F(StatsThreadLocalStoreTest, BasicSingleHistogramMerge) {
+  InSequence s;
+  store_->initializeThreading(main_thread_dispatcher_, tls_);
+
+  Histogram& h1 = store_->histogram("h1");
+  EXPECT_EQ("h1", h1.name());
+
+  std::vector<uint64_t> h1_cumulative_values;
+  std::vector<uint64_t> h2_cumulative_values;
+  std::vector<uint64_t> h1_interval_values;
+  std::vector<uint64_t> h2_interval_values;
```

Comment:
FWIW you could declare all these in one line.  Alternatively you could also make these vectors be class member vars, since you are using the same pattern in many tests.